ansible/roles/girder/tasks/main.yml

```diff
@@ -101,6 +105,15 @@
     state: present
   sudo: yes
 
+
+- name: Create directory for build
+  file: state=directory path="{{ root_dir }}/girder/clients/web/static/built"
+
+- name: Download fontello.zip
+  command: wget -O fontello.zip https://data.kitware.com/api/v1/file/57c5d1fc8d777f10f269dece/download
```

Comment:
Does this actually prevent the file from being re-downloaded at build time? If not we can set the `GIRDER_LOCAL_FONTELLO_ARCHIVE`.

Also, are we ever running `girder-install web` during this script?